typescript/web/src/components/workspaces/update-plan-banner.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, chakra, Flex, Text } from "@chakra-ui/react";
+import { WorkspaceStatus } from "@labelflow/graphql-types";
+import { useRouter } from "next/router";
+import { RiInformationFill } from "react-icons/ri";
+import { useWorkspace } from "../../hooks/use-user";
+
+const InfoIcon = chakra(RiInformationFill);
+export const UpdatePlanBanner = () => {
```

Comment:
```suggestion
export const UpgradePlanBanner = () => {
```

(don't forget to rename the file and its story too 😉 )